controllers/invitation.go

```diff
@@ -23,7 +25,7 @@ var invitation *template.Template
 
 func init() {
 	var err error
-	invitation, err = template.ParseFiles("./mail/template.invitation")
+	invitation, err = template.ParseFiles(path.Join(os.Getenv("GOPATH"), "src/github.com/coduno/api/mail/template.invitation"))
```

Comment:
Nope. It is unsafe to depend on `$GOPATH` at runtime, because once this is compiled, there is no need for a proper Go workspace. This imposes great restrictions.

Could you please elaborate why we need that in the first place?